t/cli/test_main.sh

```diff
@@ -621,3 +621,10 @@ if ! grep "keepalive_timeout 6s;" conf/nginx.conf > /dev/null; then
 fi
 
 echo "passed: found the keepalive related parameter in nginx.conf"
+
+if ! grep "real_ip_recursive off;" conf/nginx.conf > /dev/null; then
+    echo "failed: 'real_ip_recursive off;' not in nginx.conf"
+    exit 1
+fi
+
+echo "passed: disable real_ip_recursive by default in nginx.conf"
```

Comment:
Need a test to change its default value and check, like `client_max_body_size: 512m ` in my example.

Reply:
Fixed.